### DISCUSSION TITLE
Redis 6 ACL not working?

### DISCUSSION THREAD

**Discussion by winklefr** [celery/celery#7076] [Q&A] [answered]
Hello everyone,

I am using Celery 5.2.0 with Redis 6 as broker and backend. There are both on the same machine. I created an account in Redis called celery. I can see it when I run ACL LIST in redis-cli.

I use celery in a flask app. My URLs are (for broker and backend) : redis://account:password@localhost:6379/0

If I use the default account, with or without password (I tried both, and also not specifying account:password in the URL), everything works fine.

But if I use my celery account, it does not work: _kombu.exceptions.OperationalError: WRONGPASS invalid username-password pair or user is disabled._
And when I did it before specifying a password for the default account, I had this error: _redis.exceptions.ResponseError: AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?_

So it seems that celery is not taking into account the account/password put in the URLs.

Has anyone encountered that problem already?

Many thanks.

**Comment by auvipy** [comment, marked as answer]
check all your dependencies are updated

**Reply by winklefr** [reply]
Hello,

Just to confirm that after all dependencies udpates, it works well.

Regards
